src/feature/rend/rendservice.c

```diff
@@ -546,11 +546,26 @@ copy_service_on_prunning(rend_service_t *dst, rend_service_t *src)
   dst->n_intro_points_wanted = src->n_intro_points_wanted;
 }
 
+/** Close the introduction circuits of <b>s</b>. This function does not remove
+ *  the intro points from the service. */
+static void
+close_intro_circs_of_service(const rend_service_t *s)
```

Comment:
Again, we are in `rendservice.c`, the service is implied so I would go with `close_intro_circuits()`?

Reply:
Fixed in new branch.